### ISSUE TITLE
Gallery title text is black on pixel + slow mode

### ISSUE THREAD

**Issue by sethladd** [flutter/flutter#10155]
I just ran the Gallery on my Pixel, in slow mode, and the header text is black. I would expect it to be white?

```
sethladd@sethladd-macbookpro ~/Code/flutter/examples/flutter_gallery (opacity-docs) $ flutter --version
Flutter • channel unknown • unknown source
Framework • revision 49b183c502 (3 days ago) • 2017-05-14 10:09:52 -0700
Engine • revision f1bfdddfc8
Tools • Dart 1.23.0-dev.11.11
sethladd@sethladd-macbookpro ~/Code
```
![screenshot_20170517-153138](https://cloud.githubusercontent.com/assets/5479/26179066/29b38482-3b16-11e7-9b08-1070c1651677.png)

**Comment by eseidelGoogle** [comment]
@abarth made some recent changes to our status bar color setting.  Not sure if those were related to this.

**Comment by abarth** [comment]
@jason-simmons Fixed this issue.  You might need to update.

**Comment by abarth** [comment]
Fixed in e65d47d4ba33ab47b285d04d43974677b86e923e
